src/chai/ArrayManager.hpp

```diff
@@ -416,6 +416,18 @@ class ArrayManager
    */
   bool syncIfNeeded();
 
+#if defined(CHAI_ENABLE_GPU_SIMULATION_MODE)
+  /*!
+   * \brief Turn the GPU simulation mode on or off.
+   */
+  void setGPUSimMode(bool gpuSimMode) { m_gpu_sim_mode = gpuSimMode; }
```

Comment:
What's this used for?

Reply:
CARE has a layer on top of RAJA. Calling setGPUSimMode before and after the call to RAJA is the only way I could think of to use the CHAI plugin for RAJA and still provide support for the GPU simulation mode. It would happen before and after this call: https://github.com/LLNL/CARE/blob/develop/src/care/forall.h#L78. I'm hoping this is only a temporary solution as RAJA seemed open to the idea of having a GPU simulation policy (but I doubt it will happen for a while).